src/const/timer.js

```diff
@@ -40,3 +43,12 @@ export const AVOCADORO = 'AVOCADORO';
 export const SHORT_BREAK = 'SHORT_BREAK';
 export const LONG_BREAK = 'LONG_BREAK';
 export const AVOCOMBO = 'AVOCOMBO';
+
+export const TIMER_STYLE = makeStyles(theme => ({
```

Comment:
Same as `tab.js` you can put it straight into `src/components/Timer/index.jsx`.
In this way it is more consistent for all react imports to be in `.jsx` extensions only =).